templates/package/content/arch.tmpl

```diff
@@ -0,0 +1,112 @@
+{{if eq .PackageDescriptor.Package.Type "arch"}}
+	<h4 class="ui top attached header">{{.locale.Tr "packages.installation"}}</h4>
+	<div class="ui attached segment">
+		<div class="ui form">
+
+			<div class="field">
+				<label>{{svg "octicon-note"}} {{.locale.Tr "packages.arch.pacmanconf"}}</label>
+				<div class="markup"><pre class="code-block"><code>[{{.PackageDescriptor.Owner.LowerName}}.{{.RegistryHost}}]
+SigLevel = Optional TrustAll
+Server = {{AppUrl}}api/packages/{{.PackageDescriptor.Owner.LowerName}}/arch/archlinux/x86_64</code></pre></div>
```

Comment:
@KN4CK3R
I'm not sure about the best way to do this for the large number of possible combinations of architectures and distributions. If a certain package supports 2 distributions and 2 architectures, there should be 4 different links (and `any` architecture should also be treated as unique case, links for packages with `any` architecture should be generated for all possible architectures). Archlinux and x86_64 are the default options, so I set this option. Also it would be much easier to handle those parameters on client side (since client knows it's architecture and distribution).

Reply:
Or you could just leave it like this
```
Server = <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{.PackageDescriptor.Owner.Name}}/arch/$distribution/$arch"></gitea-origin-url></code></pre></div>
```

and add instructions to replace `$distribution` with value that can be taken from `.PackageDescriptor.Metadata.DistroArch` table row from the bottom of the page.

`$arch` gets replaced by `pacman` with current architecture as is documented in the [pacman.conf(5) man page](https://man.archlinux.org/man/pacman.conf.5.en#REPOSITORY_SECTIONS)

Reply:
Then just let the user decide. Debian example:
![grafik](https://github.com/go-gitea/gitea/assets/1666336/8551782f-d418-444f-a9e2-4dc2dad3edf9)

The list below is just space seperated if there is more than one value. If not every version of a package is available in every possible combination thats bad luck...